common-lib/src/main/java/com/google/cloud/tools/intellij/BasePluginInfoService.java

```diff
@@ -105,6 +105,19 @@ public boolean shouldEnable(Feature feature) {
     return false;
   }
 
+  @NotNull
+  @Override
+  public boolean isPluginInstalled(String pluginId) {
```

Comment:
I agree, `BasePluginInfoService` maybe isn't the best place for these methods.

I think your suggestion to just delete these helpers and call the PluginManager directly makes sense. This code is currently only used in one place. (These helpers made more sense a while back, when this logic was needed in a couple places.)